TagManager.php

```diff
@@ -609,6 +609,7 @@ public function getClientSideTranslationKeys(&$result)
         $result[] = 'TagManager_GettingStartedContributeTags';
         $result[] = 'TagManager_CreateNewVersionNow';
         $result[] = 'TagManager_TagManager';
+        $result[] = 'TagManager_TagManagerNew';
```

Comment:
How about
```suggestion
        $result[] = 'TagManager_MatomoTagManager';
```
?
There might be use for both versions possibly and 'New' suffix doesn't indicate well what it is.

Reply:
Done